Meta/lint-clang-format.sh

```diff
@@ -28,21 +28,21 @@ fi
 if (( ${#files[@]} )); then
     TOOLCHAIN_DIR=Toolchain/Local/clang/bin
     CLANG_FORMAT=false
-    if command -v clang-format-15 >/dev/null 2>&1 ; then
-        CLANG_FORMAT=clang-format-15
-    elif command -v brew >/dev/null 2>&1 && command -v "$(brew --prefix llvm@15)"/bin/clang-format >/dev/null 2>&1 ; then
-        CLANG_FORMAT="$(brew --prefix llvm@15)"/bin/clang-format
-    elif command -v $TOOLCHAIN_DIR/clang-format >/dev/null 2>&1 && $TOOLCHAIN_DIR/clang-format --version | grep -qF ' 15.' ; then
+    if command -v clang-format-16 >/dev/null 2>&1 ; then
+        CLANG_FORMAT=clang-format-16
+    elif command -v brew >/dev/null 2>&1 && command -v "$(brew --prefix llvm@16)"/bin/clang-format >/dev/null 2>&1 ; then
+        CLANG_FORMAT="$(brew --prefix llvm@16)"/bin/clang-format
+    elif command -v $TOOLCHAIN_DIR/clang-format >/dev/null 2>&1 && $TOOLCHAIN_DIR/clang-format --version | grep -qF ' 16.' ; then
         CLANG_FORMAT=$TOOLCHAIN_DIR/clang-format
     elif command -v clang-format >/dev/null 2>&1 ; then
         CLANG_FORMAT=clang-format
-        if ! "${CLANG_FORMAT}" --version | awk '{ if (substr($NF, 1, index($NF, ".") - 1) < 15) exit 1; }'; then
-            echo "You are using '$("${CLANG_FORMAT}" --version)', which appears to not be clang-format 15 or later."
+        if ! "${CLANG_FORMAT}" --version | awk '{ if (substr($NF, 1, index($NF, ".") - 1) < 16) exit 1; }'; then
+            echo "You are using '$("${CLANG_FORMAT}" --version)', which appears to not be clang-format 16 or later."
             echo "It is very likely that the resulting changes are not what you wanted."
         fi
     else
-        echo "clang-format-15 is not available, but C or C++ files need linting! Either skip this script, or install clang-format-15."
-        echo "(If you install a package 'clang-format', please make sure it's version 15 or later.)"
+        echo "clang-format-16 is not available, but C or C++ files need linting! Either skip this script, or install clang-format-16."
```

Comment:
can you also update 

- `.devcontainer/features/serenity/devcontainer-feature.json` to recommend llvm_version 15, 16, and trunk, default to 16
- `.devcontainer/features/serenity/install.sh` line 6 to default to LLVM_VERSION=15,  and
- `.devcontainer/devcontainer.json` to select llvm_version 16?

Reply:
Sure! done